architecture/topics/kuryr.adoc

```diff
@@ -16,9 +16,8 @@ Kuryr SDN has some requirements regarding configuration of OpenStack it will be
 using. In particular:
 
 * Minimal service set is Keystone and Neutron.
-* A LBaaS service is required. It can be either
-  link:https://docs.openstack.org/octavia/latest/[Octavia] (with all of its own
-  dependencies) or deprecated Neutron-LBaaS v2.
+* It works with
+  link:https://docs.openstack.org/octavia/latest/[Octavia].
```

Comment:
Hey, there are two more OpenStack requirements to be added to the list:
 - Trunk Ports extension must be enabled
 - Neutron must use the Open vSwitch firewall driver
(taken from https://github.com/openshift/openshift-ansible/blob/master/playbooks/openstack/configuration.md#openstack-requirements)